conf/messages.cy

```diff
@@ -239,20 +239,20 @@ deductingTax.details.content.line4.agent	=	gyda threth eisoes wedi’i didynnu g
 deductingTax.details.content.link.intro	=	Darllenwch ragor am
 deductingTax.details.content.link.text	=	dreth ar eich incwm yn y DU os ydych yn byw dramor (yn agor tab newydd)
 
-incomeFromPropertyRentals.title	=	Incwm o Eiddo wedi’u Rhentu
-incomeFromPropertyRentals.heading	=	Incwm o Eiddo wedi’u Rhentu
-incomeFromPropertyRentals.checkYourAnswersLabel	=	Incwm o eiddo wedi’u rhentu
-incomeFromPropertyRentals.change.hidden	=	Incwm o eiddo wedi’u rhentu
-incomeFromPropertyRentals.para1.individual	=	Incwm rhent yw’r rhent rydych yn ei gael oddi wrth eich tenantiaid. Mae hyn yn cynnwys unrhyw daliadau ar gyfer:
-incomeFromPropertyRentals.para1.agent	=	Incwm rhent yw’r rhent mae’ch cleient yn ei gael oddi wrth ei denantiaid. Mae hyn yn cynnwys unrhyw daliadau ar gyfer:
-incomeFromPropertyRentals.bullet1.individual	=	defnyddio dodrefn
-incomeFromPropertyRentals.bullet1.agent	=	defnyddio dodrefn
-incomeFromPropertyRentals.bullet2.individual	=	ffioedd am wasanaethau rydych yn eu darparu (megis glanhau mannau cymunedol, dŵr poeth, gwresogi ac atgyweirio)
-incomeFromPropertyRentals.bullet2.agent	=	ffioedd am wasanaethau mae’ch cleient yn eu darparu (megis glanhau mannau cymunedol, dŵr poeth, gwresogi ac atgyweirio)
-incomeFromPropertyRentals.link	=	gyfrifo’ch incwm rhent pan fyddwch yn rhoi eiddo ar osod, ar GOV.UK (yn agor tab newydd)
-incomeFromPropertyRentals.para2.individual	=	Faint oedd cyfanswm yr incwm a gawsoch o’ch holl eiddo wedi’u rhentu?
-incomeFromPropertyRentals.para2.agent	=	Faint oedd cyfanswm yr incwm a gafodd eich cleient o’i holl eiddo wedi’u rhentu?
-incomeFromPropertyRentals.deductions.hint	=	Dyma’r swm cyn i unrhyw ddidyniadau gael eu gwneud.
+propertyRentalIncome.title	=	Incwm o Eiddo wedi’u Rhentu
```

Comment:
Property rental income - Name changed, Can you mark it as *missing welsh*

Reply:
Changed to *Missing Welsh*. Thanks